back/app/controllers/web_api/v1/events_controller.rb

```diff
@@ -118,6 +118,39 @@ def event_params
     end
   end
 
+  def finder_params
+    params.tap do |p|
+      if p.key?(:ongoing_during)
+        p[:ongoing_during] = parse_date_range(p[:ongoing_during])
+      end
+    end
+  end
+
+  def parse_date_range(date_range)
+    raise ArgumentError, 'date_range must be a String' unless date_range.is_a?(String)
+
+    date_range = date_range.reverse.chomp('[').reverse.chomp(']')
+    # Ignore extra items if there are more than two. ("Be conservative in what you send,
+    # and liberal in what you accept." — at least for now)
+    start_str, end_str = date_range.split(',')
+
+    start_date = parse_date(start_str)
+    end_date = parse_date(end_str)
+    [start_date, end_date]
+  end
+
+  def parse_date(date_str)
+    date_str = date_str.strip
+    return nil if date_str.in?(['null', ''])
+
+    config_timezone.parse(date_str)
+  end
+
+  def config_timezone
```

Comment:
I will propose to move this to `AppConfiguration` during the next CDM.

Reply:
I think that's a great idea, and we should probably be using this elsewhere (e.g. project phase start/end)